### ISSUE TITLE
# Please switch to DelvCD

### ISSUE THREAD

**Issue by WesBosch** [WesBosch/ReBuff#15]
I may continue to dabble with things here, but I likely will not be doing more updates. I would strongly suggest switching over to Tischel's release of DelvCD. DelvCD is the official successor to XIVAuras and includes a full rewrite of many features you would find here in ReBuff, including an extensive rework to JobGauge data and Bars. Their release is avaliable directly through Dalamud as a Testing plugin, and is more thouroughly vetted than my code, please give it a look!

Exports from XIVAuras and ReBuff are *mostly* compatible with DelvCD via import strings, due to overhauls in how certain triggers and settings are handled, we cannot guarentee perfect compatibility. Anything involving job gauges or bars may need to be cleaned up or remade. Thank you for understanding, and thanks for being here!

**Comment by SMallick4163** [comment]
With the plugin being down, is there a way to grab an export string for our previously used auras/rebuffs so that we can try to import and adjust them as needed in DelvCD? Thanks for the work you've done on this, I'll check out DelvCD

**Comment by Krojack** [comment]
Yeah, a way to get the config to import into DelvCD would be nice. I have a good amount of them and with several rules and alighments. It would take a good day to try to redo them all from start.

**Comment by WesBosch** [comment]
I'll see what I can do. Some kind soul has already gone through the trouble of making a PR, so I'll try to deal with it tonight

**Comment by SMallick4163** [comment]
I was able to get my auras via the following method outlined in the delv discord using the XIVAuras.config in AppData\Roaming\XIVLauncher\pluginConfigs\XIVAuras. This seems to have my changes that I added when I had Rebuff it looks like.

This version includes some improvements to the config migration logic that should in theory allow users to convert their old XIVAuras config file into the new formats.

To use this follow these steps:
1. Close the game
2. Copy your backup XIVAuras.json into %APPDATA%\XIVLauncher\pluginConfigs\DelvCD\
3. Rename it to DelvCD.json
4. Re-open game

**Comment by Krojack** [comment]
Weird, I tried that earlier and it just reset the DelvDC.json file back to default. There was an update since so maybe it was something they patched in.

**Comment by WesBosch** [comment]
Oh nice, I had seen that Tischel had made that work for their fork of XIVAuras, I'm glad to hear it works for ReBuff as well. That should be the best method for now, or at least until I can get home to build the new release
